contrib/common.cpp

```diff
@@ -95,3 +95,47 @@ void onedim_fseries_kernel(BIGINT nf, FLT *fwkerhalf, SPREAD_OPTS opts)
     }
   }
 }
+
+void onedim_fseries_kernel_1sthalf(BIGINT nf, FLT *f, dcomplex *a, SPREAD_OPTS opts)
```

Comment:
Could you adapt the doc for `onedim_fseries_kernel` to this function (and the next one) so it's clear what the `f` and `a` intermediate results are?